students/jeff_shabani/session04/file_exercise.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+import os
+
+from collections import Counter
+from pathlib import Path
+
+
+
+def main():
+    if os.name == 'nt':
+        root = Path(r'D:/JRS/Python/UW/Intro_Class/students/jeff_shabani/session04')
+        src = Path(r'D:/JRS/Python/UW/Intro_Class/students/jeff_shabani/session04/files_to_copy')
+        destination = Path(r'D:/JRS/Python/UW/Intro_Class/students/jeff_shabani/session04/copied_files')
+    else:
+        root = Path('/Volumes/GASecure/JRS/Python/UW/Intro_Class/students/jeff_shabani/session04')
+        src = Path('/Volumes/GASecure/JRS/Python/UW/Intro_Class/students/jeff_shabani/session04/files_to_copy')
+        destination = ('/Volumes/GASecure/JRS/Python/UW/Intro_Class/students/jeff_shabani/session04/copied_files')
```

Comment:
you don't need all this hardcoding of file names based on operating system.  you can do the following
```
root = Path(os.getcwd())
src = Path(f"{root}/files_to_copy")
destination = Path(f"{root}/copied_files")
```

Reply:
not really sure you need the `Path()` call either

Reply:
I removed the Path call from root and destination with no negative affects. When I remove it from src I get this error "AttributeError: 'str' object has no attribute 'iterdir'". I tried different fixes using the os module and nothing helped.